lookout/style/format/analyzer.py

```diff
@@ -105,8 +107,13 @@ def group_line_nodes(y, y_pred, vnodes_y, rule_winners):
                         comment.append(comment)
                     log.warning("Failed to parse %s", file.path)
                     continue
-                X, y, vnodes_y, vnodes = res
-                y_pred, rule_winners = rules.predict(X, vnodes_y, vnodes, fe)
+                X, y, vnodes_y, vnodes, vnodes_parents, parents = res
+                y_pred, rule_winners, safe_preds = rules.predict(
```

Comment:
As I wrote in https://github.com/src-d/style-analyzer/pull/289#discussion_r230721163 we need to extract UAST filtering out of `rules.predict`